app/administrator/questions/create/(components)/typezone.tsx

```diff
@@ -0,0 +1,107 @@
+import { ReactElement, useEffect, useState } from 'react';
+import { IAnswer } from '../page';
+import { FormControl, Input } from '@mui/material';
+import {
+  PlusCircleIcon
+} from '@heroicons/react/24/outline';
+import clsx from 'clsx';
+
+interface ITypeZone {
+  questionType: string | undefined;
+}
+
+const initialAnswer = { id: 0, answer: '', isCorrect: false }
+
+export default function RenderQuestionTypeZone(props: ITypeZone): ReactElement {
+  const { questionType } = props;
+  const [answers, setAnswers] = useState<IAnswer[]>([
+    initialAnswer
+  ]);
+
+  // Notes: Reset selected answers on 'questionType' changes.
+  useEffect(() => {
+    const resetAnswersSelected = answers.map(answ => ({...answ, isCorrect: false }))
+    setAnswers(resetAnswersSelected);
+  }, [questionType]);
+
+  const handleAnswerChanges = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const modifiedAnswers = answers.map((answ: IAnswer) => {
+      if (answ.id === +event.target.id.split('-')[1]) {
+        return {
+          ...answ,
+          answer: event.target.value,
+        };
+      }
+      return answ;
+    });
+    setAnswers(modifiedAnswers);
+  };
+
+  const handleModifyAnswer = () => {
+    const modifiedAnswers = [...answers];
+    modifiedAnswers[modifiedAnswers.length - 1].id = modifiedAnswers.length;
+    modifiedAnswers.push(initialAnswer);
+    setAnswers(modifiedAnswers);
+  }
+
+  const renderAnswer = (answ: IAnswer, index: number): ReactElement => {
+    return (
+      <FormControl
+        key={`answer-${answ.id}`}
+        variant="standard"
+        className="my-2.5 inline-flex w-2/5 flex-row items-center"
+      >
+        { answ.id ? <input
```

Comment:
Thay vì input (checkbox) thì mình cùng Checkbox của Mui và thêm css đc k a?

Reply:
@sts-nghianguyen, I attempted to use MUI 5 `**_Checkbox_**` however stylistic conflicts between MUI v5 and Tailwind resulted in needless CSS adjustments. In any case, we continue to use MUI v5 to develop UI, and in some cases, we may be flexible to prevent unnecessary CSS modifications. Thanks @sts-nghianguyen